config/twojoint.yml

```diff
@@ -4,6 +4,7 @@ can_ports: ["can6"]
 
 max_current_A: 2
 has_endstop: true
+homing_at_endstop: true
```

Comment:
It might be a bit clearer what is actually happening if instead of a `homing_at_endstop` option, it would be `homing_with_index` (basically the same but negated).
The current name could be a bit confusing because in both cases it uses the end_stop if `has_endstop == true` and on the other hand it can also be a valid use-case to use it with `had_endstop == false` (meaning "simply home at the current position without moving automatically").

Reply:
Yes, you are right. Much cleaner.